bootloader/src/pyi_python.c

```diff
@@ -109,6 +110,9 @@ pyi_python_map_names(HMODULE dll, int pyvers)
     GETVAR(dll, Py_OptimizeFlag);
     GETVAR(dll, Py_VerboseFlag);
     GETVAR(dll, Py_UnbufferedStdioFlag);
+    if (pyvers >= 37) {
```

Comment:
These need to be adjusted for the new `pyvers` scheme - major*100 + minor. So 307.